common/djangoapps/student/management/commands/manage_user.py

```diff
@@ -91,6 +92,11 @@ def handle(self, username, email, is_remove, is_staff, is_superuser, groups, *ar
         self._maybe_update(user, 'is_staff', is_staff)
         self._maybe_update(user, 'is_superuser', is_superuser)
 
+        # Set unusable password if specified
+        if unusable_password and user.has_usable_password():
+            self.stderr.write(_('Setting unusable_password for user "{}"').format(user))
```

Comment:
Nitpick: this is a log message, so you can drop the hyphen here and use "unusable password."